backend_new/src/controllers/application.controller.ts

```diff
@@ -68,6 +75,23 @@ export class ApplicationController {
     return await this.applicationService.list(queryParams);
   }
 
+  @Get(`csv`)
+  @ApiOperation({
+    summary: 'Get applications as csv',
+    operationId: 'listAsCsv',
+  })
+  @Header('Content-Type', 'application/json')
+  async listAsCsv(
```

Comment:
since permissioning was brought in already, can we add the correct guards to this endpoint